tembo-cli/src/cmd/login.rs

```diff
@@ -25,14 +25,18 @@ struct TokenRequest {
 
 pub fn execute() -> Result<(), anyhow::Error> {
     let env = get_current_context()?;
-    let profile = env
-        .selected_profile
-        .as_ref()
-        .ok_or_else(|| anyhow!("Cannot log in to the local context, please select a tembo-cloud context before logging in"))?;
-    let login_url = url(profile)?;
-    let rt = tokio::runtime::Runtime::new().expect("Failed to create a runtime");
-
-    rt.block_on(handle_tokio(login_url))?;
+    if env.target == "tembo-cloud" {
+        let profile = env
+            .selected_profile
+            .as_ref()
+            .ok_or_else(|| anyhow!("Environment not setup properly"))?;
```

Comment:
Can you improve this error message so the user knows what to do and it looks nice when they see it?